docker/php.dockerfile

```diff
@@ -49,7 +49,8 @@ ENV PATH=/opt/bin:$PATH
 
 RUN test $PHP_SRC_OPCACHE = "disable" || \
         echo "zend_extension=opcache.so" > /etc/php.d/opcache.ini && \
-        echo "opcache.enable_cli=1"     >> /etc/php.d/opcache.ini
+        echo "opcache.enable_cli=1"     >> /etc/php.d/opcache.ini && \
+        echo "opcache.jit=function"     >> /etc/php.d/opcache.ini
```

Comment:
The tracing JIT breaks everything, at least for the test suite.

Users may have some success with a tracing JIT, it depends on the nature of the code. I think in the vast majority of cases function level JIT should be good enough and provide predictable results ...

Depending on how broken tracing is, we might have to force the JIT into function mode, I haven't looked very deeply into it.